net/openvswitch/files/openvswitch.init

```diff
@@ -90,29 +95,41 @@ ovs_xx() {
 	esac
 }
 
-ovs_bridge_parse_port() {
-	case "$1" in
-		*:*)
-			port="${1%%:*}"
-			type="${1#*:}"
-		;;
-		*)
-			port="$1"
-			type=""
-		;;
-	esac
-}
-
 ovs_bridge_port_add() {
-	[ -n "$1" ] || return
+	local cfg="$1"
+
+	local bridge disabled ofport port tag type
+
+	config_get_bool disabled "$cfg" disabled 0
+	[ "$disabled" = "0" ] || return
+
+	config_get bridge "$cfg" bridge
+	ovs-vsctl br-exists ovsw0 || return
```

Comment:
`ovsw0` line leftover from debug phase?

Reply:
Forgot to replace ovsw0 with $bridge, good catch!